Dockerfile

```diff
@@ -1,11 +1,20 @@
 FROM python:3.9-alpine
```

Comment:
Take a look here:
https://github.com/yaptide/yaptide/issues/115#issuecomment-964322162

this should speedup testing process

Reply:
@p1003 why did you reverted back to `alpine` flavor ?